lib/ace/autocomplete/popup.js

```diff
@@ -327,6 +327,11 @@ dom.importCssString("\
 .ace_dark.ace_editor.ace_autocomplete .ace_marker-layer .ace_active-line {\
     background-color: #3a674e;\
 }\
+.ace_editor.ace_autocomplete .ace_line .ace_in-use-field-completion:before {\
+    content: '•';\
+    padding-right: 5px;\
+    color: #1A73E8;\
+}\
```

Comment:
I wonder if we can group this either at the top of or in a second `importCssString` call so that we can more easily  pull in updates to this lib without conflicts.
A comment as to the purpose of this line _and_ that it's looker specific might be warranted to help resolve such conflicts in the future.

Reply:
Good point. Done